.github/workflows/pytest.yml

```diff
@@ -0,0 +1,192 @@
+name: Reusable WF - Run Pytest
+
+on:
+  workflow_call:
+    inputs:
+      platform:
+        required: true
+        type: string
+      platform-pretty:
+        required: true
+        type: string
+      config-fp:
+        required: false
+        type: string
+      config-pretty:
+        required: false
+        type: string
+        default: "[no config]"
+      python-version:
+        required: true
+        type: string
+      upstream-details:
+        required: false
+        type: string
+      workers:
+        required: false
+        type: string
+        default: "16"
+      pmlb-cache:
+        required: false
+        type: string
+        default: "false"
+      telemetry:
+        required: false
+        type: string
+        default: "true"
```

Comment:
You can use the built-in 'boolean' type here. :)

Actually... is this really needed? Are there times when we don't want this to run? I would say that since it is such low overhead we always have it on when it's there.